change/@azure-communication-react-254dcd24-ad35-401b-be39-ae52d3869471.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
this should be minor as we are introducing new strings Contoso may have to override/localize themselves